benchmarks/media-streaming/client/files/videoperf/stat/basic.c

```diff
@@ -202,23 +203,62 @@ perf_sample (Event_Type et, Object *obj, Any_Type reg_arg, Any_Type call_arg)
 	rate = weight*num_replies;
 
 	if (!started_dump_stats) {
-		printf("Benchmark stats...\n");
+    if(verbose)	{
+      printf("Benchmark stats...\n");
+    }
 		started_dump_stats = 1;
 	}
-	for (i=0; i < MAX_LOG_FILES; i++) {
-		printf("%d ", basic.num_active_conns[i]);
-	}
-	printf("\n");
+  if(verbose){
+    printf("\n\n\n");
+	  for (i=0; i < MAX_LOG_FILES; i++) {
+		  printf("%d ", basic.num_active_conns[i]);
+	  }
+	  printf("\n");
+  }
 #ifdef UW_THROUGHPUT_STATS
 	/* convert throughput into Mbps */
-	double throughput = weight * bytes_received / 125000.0;
-	if (verbose)
-		printf("throughput = %-8.2lf reply-rate = %-8.1f connected-conns = %-8d\n", 
-				throughput, rate, basic.num_connected_conns);
+	double throughput = weight * bytes_received * 8.0 / 1048576; // aansaarii: 1048576 = 1 M (2^20)
```

Comment:
These two calculations are the same.

Reply:
Why do we want to change the benchmark from MB to MiB? It's not measuring binary-addressable things like caches.